engine/core/src/main/java/org/datacleaner/job/builder/AnalysisJobBuilder.java

```diff
@@ -1000,7 +1014,7 @@ public void setDefaultRequirement(final ComponentRequirement defaultRequirement)
     /**
      * Gets a default requirement, which will be applied to all newly added row
      * processing components.
-     * 
+     *
```

Comment:
In the diff these single-whitespace changes look a bit weird. I can live with it, but if it's a simple thing to fix it would be nice.